doc/site/articles/yardmaps.markdown

```diff
@@ -52,10 +52,12 @@ Basic characters:
 | u      | ❌                    | ✅                   | ❌                      | ✅                     | ❌          | ❌         |
 | e      | ❌                    | ⚠️ exit only         | ❌                      | ⚠️ exit only           | ❌          | ❌         |
 
-Others:
+Remarks:
 
- * `h` has a special meaning - it **has to be the first letter and means a high resolution yardmap**,
-i.e. there have to be 4x more characters and each will correspond to a "regular" 8x8 elmo game square instead of the larger 16x16 elmo footprint square.
+ * Needs geo means the unit can only be built on top of terrain marked as geothermal.
+ * `h`: special character that when passed as (and **only valid** when) the first letter on a yardmap (e.g. `hyyy..`) marks it as high resolution: requires 4x more characters and each will correspond to a "regular" 8x8 elmo game square instead of the larger 16x16 elmo footprint square. Does not count as a square.
+ * `c`: Open and unblocking only while constructing (controlled by animation via `COB.YARD_OPEN`)
+ * `i`: Closed and blocking only while constructing (controlled by animation via `COB.YARD_OPEN`)
  * `w`, `x`, `f`:  deprecated, same as `o`.
```

Comment:
What does `stackable` mean? Does it count as a tile or just as a marker?

Reply:
It's a tile that counts as open (non-blocking) for the purpose of future buildings built on top of it, but as blocking for itself being built on top of another.

So in particular:
* you can build `o` on top of an existing `y` (because `y` is always open) and, symmetrically, you can build an `y` on top of an existing `o`.
* you can build `o` on top of an existing `s` (because `s` counts as open when it's already there), but there is no symmetry, and you CANNOT build an `s` over an existing `o` (because `s` counts as blocking otherwise).